ebos/eclequilinitializer.hh

```diff
@@ -100,7 +100,7 @@ public:
 
         EQUIL::DeckDependent::InitialStateComputer<TypeTag> initialState(materialLawManager,
                                                                          eclState,
-                                                                         vanguard.grid(),
+                                                                         vanguard,
```

Comment:
I would appreciate it if we would not pass the vanguard here, but the gridView. I seems like we only need to access the view itseld, the grid (accessible via `gridView.grid()` and the communicator (accessible via `gridView.grid().comm()`.

Reply:
I agree. I will update.